### PR TITLE
Jormungandr: add same_journey_schedules link in journey

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -303,15 +303,24 @@ def wrapper(*args, **kwargs):
...
                     journey['links'] = [create_external_link('v1.journeys', rel='journeys', **args)]
+                else:
+                    args['min_nb_journeys'] = 5
+                    ids = {o['stop_point']['stop_area']['id']
+                           for s in journey['sections'] if 'from' in s
+                           for o in (s['from'], s['to']) if 'stop_point' in o}
+                    if ids:
+                        for id in ids:
+                            args.setdefault('allowed_id[]', []).append(id)
+                        journey['links'] = [create_external_link('v1.journeys', rel='other_schedules', **args)]
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: I'm not convinced by `other_schedule` but I don't have a better thing to propose :wink:

**Review by antoine-de — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 2]
```diff
@@ -303,15 +303,23 @@ def wrapper(*args, **kwargs):
...
                     if 'from' not in args:
                         args['from'] = journey['from']['id']
-                    if 'region' in kwargs:
-                        args['region'] = kwargs['region']
                     journey['links'] = [create_external_link('v1.journeys', rel='journeys', **args)]
+                elif ids and 'public_transport' in (s['type'] for s in journey['sections']):
+                    args['min_nb_journeys'] = 5
+                    ids.update(args.get('allowed_id[]', []))
+                    args['allowed_id[]'] = list(ids)
+                    journey['links'] = [create_external_link('v1.journeys', rel='other_schedules', **args)]
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: voting for `same_journey_schedules` here... any preference @stifoon ?

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py (hunk 2 above)
Comment (stifoon) [reply source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: I would love `same_journey_shoot_again`

but I know you are not so funny. So you can use `same_journey_schedules`

Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: So @stifoon voted "same_journey_schedule" when pressed with only those 2 options :)

**Pull request merged**